packages/trader/src/Modules/Reports/Containers/open-positions.jsx

```diff
@@ -444,6 +439,7 @@ export default connect(({ modules, client, common, ui }) => ({
     onClickCancel: modules.portfolio.onClickCancel,
     onClickSell: modules.portfolio.onClickSell,
     onMount: modules.portfolio.onMount,
-    onUnmount: modules.portfolio.onUnmount,
     server_time: common.server_time,
 }))(withRouter(OpenPositions));
+
+module.exports = OpenPositions;
```

Comment:
Why? it is being exported as the default!

Reply:
This export is required for tests